utils/vscode/package.json

```diff
@@ -33,8 +39,19 @@
         "scopeName": "source.carbon",
         "path": "./carbon.tmLanguage.json"
       }
+    ],
+    "snippets": [
+      {
+        "language": "carbon",
+        "path": "./carbon.code-snippets"
+      }
     ]
   },
+  "keywords": [
+    "carbon",
+    "snippet",
+    "snippets"
```

Comment:
We're apparently limited to 5 keywords, so I don't know if we should spend 2 of them on "snippet". That said, there's probably not much point in having this field at all until we publish the extension.

Reply:
Ok, I will remove one